scripts/push_and_create_pr.sh

```diff
@@ -25,7 +23,7 @@ create_pr(){
             --head $branch \
             | cut -d "/"  -f 7)
 
-        if [ $APPROVE_LABEL == "auto-approve" ]; then
+        if [[ -n $APPROVE_LABEL ]] && [ $APPROVE_LABEL == "auto-approve" ]; then
```

Comment:
We could maybe skip checking what the label contains if we only set it to auto-approve?